cityofaustin/atd-moped
# Disassociate child projects when parent is deleted

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9329

## Testing
**URL to test:** 

https://9329-disassociate-subprojects--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

1. Add subprojects to a project (since https://github.com/cityofaustin/atd-moped/pull/696 isnt merged yet, the only way to see the relationship in the UI right now is via the parent project page) Remember the project_id of the project. 
2. Delete the project (three dots under the following icon, select delete)
3. Moped will redirect you to the projects page. Navigate back to the deleted project's page (using the project_id you remembered in step 1), the subprojects table should be empty. The only indication the project has been deleted is you can't see it on the projects list table. 


---
#### Ship list
